modules/stream/src/main/scala/ch/epfl/bluebrain/nexus/sourcing/persistence/IndexerConfig.scala

```diff
@@ -92,6 +96,8 @@ object IndexerConfig {
       name: Option[String] = None,
       mapping: Option[Event => F[Option[MappedEvt]]] = None,
       index: Option[List[MappedEvt] => F[Unit]] = None,
+      mapInitialProgress: Option[ProjectionProgress => F[Unit]] = None,
```

Comment:
If in the end when building IndexerConfig you are defaulting to `_ => F.unit`, why not to do it here already:
```scala
mapInitialProgress: Option[ProjectionProgress => F[Unit]] = Some(_ => F.unit)
```